server/controller/assessment/getTableData.ts

```diff
@@ -13,32 +14,28 @@ export const getTableData = async (
     tableNames: Array<string>
   },
   client: BaseProtocol = DB
-): Promise<Array<{ tableName: string; data: TableData }>> => {
+): Promise<TableData> => {
   const { countryIso, assessmentName, cycleName, section, tableNames } = props
   if (!section) throw new Error('missing section')
 
+  const tables: Record<string, any> = {}
+  tableNames.forEach((tableName) => {
+    tables[tableName] = {}
+  })
+
   return client.tx(async (t) => {
     const assessment = await AssessmentRepository.read({ name: assessmentName }, t)
+    const cycle = assessment.cycles.find((cycle) => cycle.name === cycleName)
 
-    const tables = await Promise.all(
-      tableNames.map(async (tableName) => {
-        const { data } = await AssessmentRepository.readTableData(
-          {
-            assessment,
-            countryIso,
-            cycleName,
-            tableName,
-          },
-          t
-        )
+    return TableDataRepository.getTableData(
```

Comment:
remove TableDataRepository and use CycleDataRepository

Reply:
Rename